includes/Classifai/Features/Classification.php

```diff
@@ -458,7 +458,7 @@ public function render_meta_box( \WP_Post $post ) {
 			</label>
 		</p>
 
-		<div class="classifai-clasify-post-wrapper" style="display: none;">
+		<div class="classifai-classify-post-wrapper" style="display: none;">
```

Comment:
While this is a typo, because this is used as a class name, we'll need to decide if we want to leave this as-is, as others may be relying on this class for styling purposes and updating this will break things for them. Not overly concerned but something for us to consider

Reply:
Just adding additional details here that this is used in the classic editor only.